archetypes/Browse/FilterSelects/Bar.tsx

```diff
@@ -1,15 +1,33 @@
+import React, { useEffect, useState } from 'react';
 import { Dropdown } from '@components/General';
 import { SearchInput } from '@components/General/SearchInput';
-import React from 'react';
 import { BrowseAction, BrowseState, LeverageFilterEnum, SideFilterEnum, SortByEnum } from '../state';
 import TooltipSelector, { TooltipKeys } from '@components/Tooltips/TooltipSelector';
+import useBrowsePools from '@libs/hooks/useBrowsePools';
 
 interface FilterSelectsProps {
     state: BrowseState;
     dispatch: React.Dispatch<BrowseAction>;
 }
 
 const FilterSelects: React.FC<FilterSelectsProps> = ({ state, dispatch }) => {
+    const [leverageState, setLeverageState] = useState<string | null>();
+    const [sideState, setSideState] = useState<string | null>();
+    const [sortByState, setSortByState] = useState<string | null>();
+    useEffect(() => {
+        if (sessionStorage.getItem('setLeverage') !== null) {
+            setLeverageState(sessionStorage.getItem('setLeverage'));
+            dispatch({ type: 'setLeverage', leverage: leverageState as LeverageFilterEnum });
+        }
+        if (sessionStorage.getItem('setSide') !== null) {
+            setSideState(sessionStorage.getItem('setSide'));
+            dispatch({ type: 'setSide', side: sideState as SideFilterEnum });
+        }
+        if (sessionStorage.getItem('setSortBy') !== null) {
+            setSortByState(sessionStorage.getItem('setSortBy'));
+            dispatch({ type: 'setSortBy', sortBy: sortByState as SortByEnum });
+        }
+    }, [useBrowsePools()]);
```

Comment:
This seems weird. What was the reasoning behind putting this hook here??. This is for loading the pools.